README.md

```diff
@@ -9,18 +9,25 @@ There is also a downloadable [reference manual](https://rit-vex-u.github.io/Core
 
 ## Getting Started
 
+
 The recommended way to use this repo is through [git subtree](https://git-memo.readthedocs.io/en/latest/subtree.html).
 
-To setup core for a new project
+To setup core for a new project:
 1. Create a new vex project (using the VSCode extension or other methods)
 2. Initialize a git repository for the project
 3. Execute `git subtree add --prefix=core git@github.com:RIT-VEX-U/Core.git main`
 4. Update the vex Makefile (or any other build system) to know about the core files (`core/src` for source files, `core/include` for headers)
 5. Start coding :)
 
+If you require any features that make use of [Eigen](https://eigen.tuxfamily.org/index.php?title=Main_Page), you can do the following.
```

Comment:
don't all projects that use core versions that include n pod or anything in the future that requires eigen require updating if they update core regardless if the project itself uses it?

Reply:
i see you resolved this but i did realize that technically, you dont need eigen if you never include n pod bc its all headers. however, like thats already an if and as soon as we start using eigen in .cpp files it wont be true anymore so im gonna update that